index/scorch/scorch.go

```diff
@@ -36,6 +36,8 @@ const Version uint8 = 2
 
 var ErrClosed = fmt.Errorf("scorch closed")
 
+var MappingInternalKey = []byte("_mapping")
```

Comment:
Should make this a `const` - and I don't think this is the right place for this as @CascadingRadium commented earlier.